frontend/src/components/Pages/RobotPage/RobotArmMovement.tsx

```diff
@@ -1,23 +1,42 @@
 import { Button } from '@equinor/eds-core-react'
-import { useState } from 'react'
+import { useState, useEffect } from 'react'
 import { BackendAPICaller } from 'api/ApiCaller'
 import { Robot } from 'models/Robot'
 import { TranslateText } from 'components/Contexts/LanguageContext'
 
 interface RobotProps {
     robot: Robot
     armPosition: string
+    buttonActivated?: boolean
```

Comment:
buttonActivated can be interpreted as button has been clicked/activated. Consider different name.

Possibly name isRobotAvailable?